src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,32 @@
+import { useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
```

Comment:
The initial state for `people` is set to an empty array, which is correct. However, ensure that the `Person` type is correctly defined in `../types` to avoid type errors.